docs/content/error/$http/baddata.ngdoc

```diff
@@ -0,0 +1,16 @@
+@ngdoc error
+@name $http:baddata
+@fullName Bad JSON Data
+@description
+
+The default @{link ng$http#default-transformations `transformResponse`} will try to parse the
+response as JSON if the `Content-Type` header is `application/json` or the response looks like a
+valid JSON-stringified object or array.
+This error occurs when that data is not a valid JSON object.
+
+The error message should provide additional context such as the actual response.
+
+To resolve this error, make sure you pass a valid JSON data object to `transformResponse`.
```

Comment:
How about

To resolve this error, make sure you pass valid JSON data to `transformResponse` or use the appropriate `Content-Type` instead of the specified `application/json` if passing non-JSON data.